### DISCUSSION TITLE
Progress isn't being saved on the server

### DISCUSSION THREAD

**Discussion by Oldman-BR** [Vdauphin/HeartsAndMinds#1673] [Question and answer]
We're using the H&M in our dedicated server using the software Faster. Is all set, the option that make the mission work as a persistent is checked and the option of the line #38 of the file param.hpp (Load the savegame (if available)) is enabled, and the mission isn't saving the progress. I tested moving one of our vehicles (there's no respawn module in the mission), saved the mission as admin thru the Ace, and restarting the server, the vehicle was back to the original position.

Can you give me some advise to make the progress be saved? Thanks in advance.

**PLEASE DISREGARD THIS QUESTION. PROBLEM SOLVED. THANKS A LOT.**

**Comment by AshTrayBY** [comment]
have the same issue. @Oldman-BR can you please share your experiense
